bundle/src/test/java/com/adobe/acs/commons/ccvar/filter/ContentVariableJsonFilterTest.java

```diff
@@ -0,0 +1,322 @@
+        String before = "{\"jcr:primaryType\":\"nt:unstructured\",\"fileReference\":\"/content/dam/we-retail/en/experiences/arctic-surfing-in-lofoten/surfer-wave-01.jpg\",\"useFullWidth\":\"true\",\"title\":\"{{page_properties.jcr:title}}\",\"sling:resourceType\":\"weretail/components/content/heroimage\"}";
+        doAnswer(invocation -> {
+            HttpServletResponse response =
+                    (HttpServletResponse) invocation.getArguments()[1];
+            response.setCharacterEncoding("utf-8");
+            response.setContentType("application/json");
+            response.getWriter().println(before);
+            return null;
+        }).when(mocked).doFilter(any(), any());
+
+        String after = "{\"jcr:primaryType\":\"nt:unstructured\",\"fileReference\":\"/content/dam/we-retail/en/experiences/arctic-surfing-in-lofoten/surfer-wave-01.jpg\",\"useFullWidth\":\"true\",\"title\":\"Arctic Surfing In Lofoten\",\"sling:resourceType\":\"weretail/components/content/heroimage\"}";
+        filter.doFilter(context.request(), context.response(), mocked);
+        assertTrue(startsWith(after)
+                .matches(context.response().getOutputAsString()));
+    }
+
+    @Test
+    public void testInvalidConfig() throws IOException, ServletException {
+        Map<String, Object> config = new HashMap<>();
+        config.put("includes", new String[]{"*.model*"});
+        config.put("excludes", new String[]{"*model*"});
+        initServices(config);
+        context.currentResource("/content/we-retail/language-masters/en/experience/arctic-surfing-in-lofoten/jcr:content/root/hero_image");
+        context.request().setResource(context.currentResource());
+        context.request().setPathInfo("/content/we-retail/language-masters/en/experience/arctic-surfing-in-lofoten/jcr:content/root/hero_image.json");
+        context.requestPathInfo().setExtension("json");
+
+        FilterChain mocked = mock(MockFilterChain.class);
+        String before = "{\"jcr:primaryType\":\"nt:unstructured\",\"fileReference\":\"/content/dam/we-retail/en/experiences/arctic-surfing-in-lofoten/surfer-wave-01.jpg\",\"useFullWidth\":\"true\",\"title\":\"{{page_properties.jcr:title}}\",\"sling:resourceType\":\"weretail/components/content/heroimage\"}";
+        doAnswer(invocation -> {
+            HttpServletResponse response =
+                    (HttpServletResponse) invocation.getArguments()[1];
+            response.setCharacterEncoding("utf-8");
+            response.setContentType("application/json");
+            response.getWriter().println(before);
+            return null;
+        }).when(mocked).doFilter(any(), any());
+
+        String after = "{\"jcr:primaryType\":\"nt:unstructured\",\"fileReference\":\"/content/dam/we-retail/en/experiences/arctic-surfing-in-lofoten/surfer-wave-01.jpg\",\"useFullWidth\":\"true\",\"title\":\"Arctic Surfing In Lofoten\",\"sling:resourceType\":\"weretail/components/content/heroimage\"}";
```

Comment:
i view both include and exclude rules as 'limiters' from the default of 'all'. I don't love the idea of making people add '.*' to this config to have it run on all JSON output (like it could potentially be used on a publish instance), but i can agree with (yet another) 'limit first' approach for configuration of the tool. i'll update this

Reply:
I was actually not clear on my statement.

Im actually fine with a default include being used in the absence of a configuration.  However, if an admin does configure an "include" list then we honor that list.  And if the only item in that list is invalid (and thus matches nothing), we should treat it the same as an include list with a single valid entry that also matches nothing.

Reply:
Ah okay that makes sense - I don't love the way I had to handle it, but I've updated the PR